server/fleet/vpp.go

```diff
@@ -35,6 +35,8 @@ type VPPAppTeam struct {
 	// ValidatedLabels are the labels (either include or exclude any) that have been validated by
 	// Fleet as being valid labels. This field is only used internally.
 	ValidatedLabels *LabelIdentsWithScope `json:"-"`
+	// AddedAt is when the VPP app was added to the team
+	AddedAt time.Time `db:"added_at" json:"created_at"`
```

Comment:
is this field to indicate that this is the vpp app -> team relation vs an underlying entity?

Reply:
Yep, and to get this out of the way of queries that contain `vpp_apps.created_at` as well